src/endpoints/brands/brands.types.ts

```diff
@@ -0,0 +1,3 @@
+export type * from './brands'
```

Comment:
@lowbits `export type *` doesn't work with our current TS version, so please change them to named exports.

That way we also only export what we actually want

Reply:
I fixed it, its hard to know which types we need to export, because shops which use the bapi/sdk can need other types. Maybe would be a nice idea to update typescript, in fact how many shops use this now.